cornice/tests/validationapp.py

```diff
@@ -155,13 +155,23 @@ def post7(request):
         null
     )
 
-    from cornice.validators import colander_validator
+    from cornice.validators import colander_validator, colander_body_validator
 
     COLANDER = True
 except ImportError:
     COLANDER = False
 
 if COLANDER:
```

Comment:
`<msg mode="getshitdone">` Please do it in another PR, this is out of scope here `</msg>`